api/test/wfl/tools/endpoints.clj

```diff
@@ -41,9 +41,9 @@
 
 (defn get-workflows
   "Query v1 api for all workflows managed by `_workload`."
-  [{:keys [uuid] :as _workload}]
+  [{:keys [uuid] :as _workload} & [status]]
   (-> (wfl-url "/api/v1/workload/" uuid "/workflows")
-      (http/get {:headers (auth/get-auth-header)})
+      (http/get (merge {:headers (auth/get-auth-header)} (when status {:query-params {:status status}})))
       util/response-body-json))
```

Comment:
Maybe just:
``` clojure
(defn get-workflows
  "Query v1 api for all workflows managed by `_workload`."
  [{:keys [uuid] :as _workload} & [status]]
  (-> (wfl-url "api" "v1" "workload" uuid "workflows")
      (http/get (merge {:headers (auth/get-auth-header)}
                       (when status {:query-params {:status status}})))
      util/response-body-json))
```

BTW: I like the `(merge … (when …))`. It never occurs to me to do that.